src/main/java/com/answerdigital/answerking/repository/CategoryRepository.java

```diff
@@ -1,12 +1,13 @@
 package com.answerdigital.answerking.repository;
 
 import com.answerdigital.answerking.model.Category;
+import org.springframework.data.repository.CrudRepository;
 import org.springframework.data.repository.PagingAndSortingRepository;
 import org.springframework.stereotype.Repository;
 import java.util.Set;
 
 @Repository
-public interface CategoryRepository extends PagingAndSortingRepository<Category, Long> {
+public interface CategoryRepository extends PagingAndSortingRepository<Category, Long>, CrudRepository<Category, Long> {
```

Comment:
Would it be better to extend JpaRepository?

```suggestion
public interface CategoryRepository extends JpaRepository<Category, Long> {
```

Reply:
JpaRepository doesn't return sets (We're using Set<Category> here) because JpaRepository is extending the new List repositories.

Reply:
Got it, thanks! 😄